cypress/tests/sample.spec.ts

```diff
@@ -0,0 +1,10 @@
+// <reference types="cypress" />
+
+describe('example to-do app', () => {
```

Comment:
@florkbr Sure, we can discuss that. In this case we need both the sample plugin and the sample app to be running for the E2E tests and the tests also rely on both these packages since the plugin is integrated into the app. We may also need a script to start those servers up first. I went with the recommended directory structure from cypress as an initial step but we can change it up as needed.